### PR TITLE
All the things Rod really want for Xmas

### PR DESCRIPTION
**Pull request by rod11**

### CODE REVIEW

**Comment by rod11** [comment]
Resolves #25

File: rod/xmas-list.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- [ ] Sleep
+- [ ] More sleep
+- [ ] Rest
```
Comment (rod11) [inline rod/xmas-list.md:3]: Yes .... very good

Comment (rod11) [inline rod/xmas-list.md:3]: OK

File: rod/xmas-list.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+- [ ] Zero inbox
+- [ ] Empty todo list
+
+
```
Comment (rod11) [inline rod/xmas-list.md:7]: Why is this line empty?

**Comment by rod11** [comment]
Now I see how this works

File: rod/xmas-list.md (hunk 2 above)
Comment (EmilyUna65) [inline rod/xmas-list.md:7]: I think you're more likely to get a zero inbox...

**Comment by AMZod** [comment]
Sleep would be nice :sleeping:

**Comment by cowperthwait** [comment]
lgtm; 🐑

**Comment by watridge** [comment]
good list :sheep:

**Comment by monicanpm** [comment]
:sheep: it

**Pull request merged**